pkg/scheduler/eventhandlers_test.go

```diff
@@ -105,13 +106,13 @@ func TestSkipPodUpdate(t *testing.T) {
 	}
 	for _, test := range table {
 		t.Run(test.name, func(t *testing.T) {
-			c := NewFromConfig(&Config{
+			metrics.Register()
```

Comment:
is this one (in the test) actually needed?

Reply:
I am not really sure, just added it to be safe. However when removing it, the test is still successful.

Reply:
metrics is something usually only makes sense during runtime. Unless you are testing the metrics themselves. It doesn't look like that is the case here.

Reply:
Okay, done!